.github/workflows/build.yml

```diff
@@ -13,4 +13,4 @@ jobs:
       - name: Install dependencies
         run: npm ci
       - name: Build
-        run: npm run build
+        run: tsc -p tsconfig.json
```

Comment:
Would it be appropriate to rename this workflow to something else or is build still the best verb for what this CI is doing?

Reply:
I gave that some brief thought while I was working on this PR, and my initial lazy response was "no". But the fact that you also raised that question makes me revisit! Options include "Check types" and "Compile"; any others spring to mind?

Somewhat relevant is the [documentation on the `--no-emit` option](https://www.typescriptlang.org/tsconfig/#noEmit):

> Do not emit compiler output files like JavaScript source code, source-maps or declarations. [...] You can then use TypeScript [...] as a source code type-checker.

Reply:
I went with "check types"; let me know if you have strong opinions in another direction!